src/Settings.php

```diff
@@ -4,229 +4,233 @@
 
 use Tribe__Settings_Manager;
 
-/**
- * Do the Settings.
- */
-class Settings {
-
-	/**
-	 * The Settings Helper class.
-	 *
-	 * @var Settings_Helper
-	 */
-	protected $settings_helper;
-
-	/**
-	 * The prefix for our settings keys.
-	 *
-	 * Gets set automatically from the Text Domain or can be set manually.
-	 * The prefix should not end with underscore `_`.
-	 *
-	 * @var string
-	 */
-	private $opts_prefix = '';
-
+if ( ! class_exists( Settings::class ) ) {
```

Comment:
You shouldn't use the class to check if the class exists.

Reply:
That's using the special _class_ constant:
* https://www.php.net/manual/en/language.oop5.constants.php
* 1min demo this code isn't a problem: https://share.getcloudapp.com/nOuNWQoY